src/v0/holidays.ts

```diff
@@ -0,0 +1,81 @@
+import { Client } from '../client'
```

Comment:
Is there an easy way to check if this works before merging it?

Reply:
yep
1. `npm run build` for tillhub-sdk-javascript
2. Link tillhub-sdk-javascript repo folder to node_modules/@tillhub/javascript-sdk of the targeted project using symlink
3. Start the targeted project